PWGHF/vertexingHF/vHFML/AliAnalysisTaskSEDs.cxx

```diff
@@ -1669,8 +1672,10 @@ void AliAnalysisTaskSEDs::CreateCutVarsAndEffSparses()
   Double_t maxMass = massDs + 0.5 * nInvMassBins * fMassBinSize;
 
   Int_t nSparseAxes = knVarForSparse;
-  if (!fApplyML)
-    nSparseAxes--;
+  if (!fApplyML && !fMultiClass)
+    nSparseAxes -= 3;
+  if (fApplyML && !fMultiClass)
+    nSparseAxes -= 2;
   if (fApplyML && fUseMinimalVarForSparse)
     nSparseAxes = knVarForSparseMLMinimal;
```

Comment:
Now, the "minimal" configuration of the THnSparse has always 5 axes, right? We should have 3 for binary classification.